hadoop-ozone/tools/src/test/java/org/apache/hadoop/ozone/repair/om/TestTransactionInfoRepair.java

```diff
@@ -125,12 +126,14 @@ private void testCommand(ManagedRocksDB mdb, ColumnFamilyHandle columnFamilyHand
           .thenReturn(transactionInfo2);
 
       CommandLine cli = new OzoneRepair().getCmd();
-      cli.execute(
-          "om",
-          "update-transaction",
-          "--db", DB_PATH,
-          "--term", String.valueOf(TEST_TERM),
-          "--index", String.valueOf(TEST_INDEX));
+      withTextFromSystemIn("y")
```

Comment:
Can we do this without adding a new dependency? It does flow nicely but it might be better to avoid adding small/niche libraries that are rarely needed just for syntactic sugar.

Reply:
It's for test only.

Adding as dependency and copying/reimplementing it both have benefits and drawbacks.  E.g.:

- (+) It has test coverage that I will not add if I need to implement it from scratch.
- (-) I wish it would allow lambdas with return type, to avoid the need for the `AtomicInteger` for catching exit code.